docs/learn-governance.md

```diff
@@ -261,20 +261,22 @@ against - equate to a simple majority-carries system at 100% turnout.
 
 ## Council
 
-To represent passive stakeholders, we introduce the idea of a "council". The council is an on-chain
-entity comprising a number of actors, each represented as an on-chain account. For Polkadot this
-number is likely to begin at around six people, and increase over the course of 9 months to 24
-people (roughly one extra individual coming on every two weeks). In general, it has a fixed number
-of seats (24 on Polkadot and 17 on Kusama).
+To represent passive stakeholders, Polkadot introduces the idea of a "council". The council is an
+on-chain entity comprising a number of actors, each represented as an on-chain account. On Polkadot
+the council currently consists of 13 members, however this is expected to increase over the next few
+months to 24 people. In general, the council will end up having a fixed number of seats. On
+Polkadot, this will be 24 seats while on Kusama it is 17 seats.
 
-The council is called upon primarily for two tasks of governance: proposing sensible referenda, and
-cancelling uncontroversially dangerous or malicious referenda.
+The council is called upon primarily for three tasks of governance: proposing sensible referenda,
+cancelling uncontroversially dangerous or malicious referenda, and electing the technical committee.
 
 For a referendum to be proposed by the council, a strict majority of members must be in favor, with
 no member exercising a veto. Vetoes may be exercised only once by a member for any single proposal;
-if, after a cool-down period, the proposal is resubmitted, they may not veto it a second time. In
-the case that all members vote in favor, the vote is considered unanimous and is treated as
-uncontroversial.
+if, after a cool-down period, the proposal is resubmitted, they may not veto it a second time.
+Council motions which pass with a strict majority - but without reaching unanimous support will move
+to public referendum under a neutral, majority-carries adaptive quorum biasing. In the case that all
```

Comment:
```suggestion
to public referendum under a neutral, majority-carries voting scheme. In the case that all
```

AQB is only when we need supermajority relative to turnout, no?

Reply:
Yes. I was using AQB here as the catch-all for however the voting scheme ends up. Strictly speaking, it only does apply when the supermajority is required relative to turnout.